SU-SWS/stanford_components
# EARTH-522 Moved close to after the links

# READY FOR REVIEW

# Summary
- Moved close to after links so the user goes to that after tabbing through the links.

# Needed By (Date)
- End of sprint

# Urgency
- Low

# Steps to Test

1. Checkout this branch and same branch in [matson](https://github.com/stanford-earth/matson/pull/86)
2. clear cache
3. review page /patterns/section_expandable_banner to make sure it looks the same and the whole block is clickable.

# Associated Issues and/or People
- https://github.com/stanford-earth/matson/pull/86

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
